src/index.ts

```diff
@@ -485,6 +487,142 @@ export function apply(ctx: Context, config: Config) {
       }
     })
 
+  const cmdLsEx = ctx.command('lsmd <modelName:text>')
+    .shortcut('lsckpt', { i18n: true, fuzzy: true, options: { ckpt: true } })
+    .shortcut('lslora', { i18n: true, fuzzy: true, options: { lora: true } })
+    .shortcut('lsemb', { i18n: true, fuzzy: true, options: { embedding: true } })
+    .shortcut('lshn', { i18n: true, fuzzy: true, options: { hypernetwork: true } })
```

Comment:
Generally the short of hypernetwork should be `hyp`, although it is not widely used.